ingest-app/src/cmr/ingest/config.clj

```diff
@@ -5,6 +5,11 @@
             [cmr.oracle.connection :as conn]
             [cmr.message-queue.config :as rmq-conf]))
 
+(defconfig ingest-accept-umm-version
```

Comment:
We want a config for this so we can change it for testing purposes. Generally the idea is to block end-users from ingesting in-development versions of UMM so we do not invalidate their collections as we make schema changes. To test our changes, however, we'd have to allow ingest of the current in-development version.

Also, the accept version is not necessarily current - 1. Right now they are the same. If we were to make schema changes this PI, current would go to 1.10 and ingest would stay 1.9.